### DISCUSSION TITLE
How to solve "numpy" error when I run App in dist folder after build

### DISCUSSION THREAD

**Discussion by theangkko** [Dadangdut33/Screen-Translate#22] [Q&A] [answered]
Thanks for wonderful App, first!

To exercise to build tkinter App, I forked your repo. 
And modified (move button and adjust width of window) a little for my using. With "python main.py", all is working and no problem. 

But, when I build '.exe' file using "python build_pyinstaller.py" then after run "ScreenTranslate.exe" in ScreenTranslate 2.1.1 folder, I got the ERROR message below.

![image](https://github.com/Dadangdut33/Screen-Translate/assets/75212211/ad5b36b8-6872-419e-8623-0c16a4c29021)

<br><br>
I have some experience with build tkinter app and use pyinstaller. I tried to do something in "pyenv" and several version of (python 3.11 and python 3.10). But I had not solved this error. If you know about this, please help me to solve.

my forked REPO is (https://github.com/theangkko/Screen-Translate-fork)

<br><br>
![image](https://github.com/Dadangdut33/Screen-Translate/assets/75212211/f5f6cc1a-0754-48fb-81ab-c979a3b654e3)

**Comment by theangkko** [comment, marked as answer]
I solved with re-install of numpy
"pip install --upgrade --force-reinstall numpy"

Thanks!!

**Reply by Dadangdut33** [reply]
Nice to see your fork, i havent had the chance to develop this again, thanks!
